python/integration_tests/test_upkeep_dlq.py

```diff
@@ -243,7 +243,7 @@ def test_upkeep_dlq() -> None:
     num_messages = 10_000
     num_partitions = 4
     max_pending_count = 100_000
-    taskbroker_timeout = 120  # the time in seconds to wait for taskbroker to process
+    taskbroker_timeout = 600  # the time in seconds to wait for taskbroker to process
```

Comment:
If we consistently go over 5min, we should consider reducing the number of messages.